flink-connector-pulsar/src/test/java/org/apache/flink/connector/pulsar/sink/writer/PulsarWriterTest.java

```diff
@@ -205,6 +207,25 @@ public int getAttemptNumber() {
             return 0;
         }
 
+        // The following three methods are for compatibility with
+        // https://github.com/apache/flink/commit/4f5b2fb5736f5a1c098a7dc1d448a879f36f801b
+        // . Removed the commented out `@Override` when we move to 1.18.
+
```

Comment:
I think we can just add the `@Override` annotation. Because this is just a test class.

Reply:
I remember that `javac` will fail on compilation when we build with Flink not `1.18-SNAPSHOT`.

Reply:
That's true. But the master branch should always focus on the latest snapshot version of Flink.